packages/api/src/webhooks/cannon.ts

```diff
@@ -392,8 +393,7 @@ export default class WebhookCannon {
 
       // sign payload if there is a webhook secret
       if (webhook.sharedSecret) {
-        let signature = sign(params.body, webhook.sharedSecret);
-        params.headers[SIGNATURE_HEADER] = `t=${timestamp},v1=${signature}`;
+        params = await addSignatureHeader(params, webhook, timestamp);
       }
```

Comment:
Related to the suggestion on the helper below, this would look like:
```suggestion
        const sigHeaders = signatureHeaders(params.body, webhook.sharedSecret, timestamp);
        params.headers = { ...params.headers, ...sigHeaders};
```